.github/workflows/flake.yml

```diff
@@ -35,6 +35,6 @@ jobs:
     - name: Lint with flake8
       run: |
         pip install flake8
-        flake8 --max-line-length 95 archivy
+        flake8 --max-line-length 100 archivy
```

Comment:
Preferred line lengths, are `79`, `99`, `119`; `100` is even and does not end with `9`